src/mbgl/style/layers/background_layer_properties.hpp

```diff
@@ -17,7 +17,7 @@ class BackgroundPaintProperties {
     void cascade(const CascadeParameters&);
     bool recalculate(const CalculationParameters&);
 
-    PaintProperty<Color> backgroundColor { { 0, 0, 0, 1 } };
+    PaintProperty<Color> backgroundColor { Color::black() };
```

Comment:
Sorry I didn't paid attention to the disclaimer on the first line. Will update `layer_properties.hpp.ejs` file accordingly.